src/sentry/plugins/sentry_mail/models.py

```diff
@@ -123,6 +126,53 @@ def get_send_to(self, project):
             logger.debug('Tried to send notification to invalid project: %r', project)
             return []
 
+        if event:
+            owners, _ = ProjectOwnership.get_owners(project.id, event.data)
+            if owners != ProjectOwnership.Everyone:
+                if not owners:
+                    metrics.incr(
+                        'features.owners.send_to',
+                        tags={
+                            'organization': project.organization_id,
+                            'outcome': 'empty',
+                        },
+                        skip_internal=True,
+                    )
+                    return []
+
+                metrics.incr(
+                    'features.owners.send_to',
+                    tags={
+                        'organization': project.organization_id,
+                        'outcome': 'match',
+                    },
+                    skip_internal=True,
+                )
+                send_to_list = []
```

Comment:
The last little bit is we should make this a `set()` just so we don't end up returning dupes.

It can be a set here, then return it as a list by coercing it back.